app/components/forms/register-form.js

```diff
@@ -55,6 +59,39 @@ export default Component.extend(FormMixin, {
         this.set('isLoading', true);
         this.sendAction('submit');
       });
+    },
+
+    showPasswordSignupConfirm() {
+      function show() {
+        $('#pwdConfirm').attr('type',  'text');
```

Comment:
Please do not use jquery for this. Just change attribute on the template based on property on the component. 

-- 

Eg. if a property `showPassword` is true, set attribute as text else password. And when they press on it use `this.toggleProperty` to toggle the value of it.